CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Add OpenAPI schema for AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+- Support secure connections to ClickHouse DB with TLS, and allow user to define custom HTTP/HTTPs url and CA certificate. ([#5171](https://github.com/antrea-io/antrea/pull/5171), [@yuntanghsu])
```

Comment:
"Add support of TLS, HTTP, and HTTPs protocols for Flow Aggregator to connect to the ClickHouse DB, and allow users to specify the CA certificate for TLS and HTTPs."